framework/src/modules/dpos_v2/utils.ts

```diff
@@ -261,3 +266,16 @@ export const getDelegateWeight = (
 	}
 	return totalVotesReceived;
 };
+
+export const isSharingCoEfficientSorted = (
```

Comment:
```suggestion
export const isSharingCoefficientSorted = (
```

Reply:
Not updating since it will be solved by #7747

Reply:
I don't see how is #7747 related to this... 🤔

Reply:
ah i thought you fixed the typo

Reply:
I did, but that was changing `Cofficient` into `Coefficient` in a code that was already approved and merged.

This is about changing `CoEfficient` into `Coefficient` [one word instead of two], and I didn't change this one since it was in an active PR.